### PR TITLE
Added a pprz time to the images

### PR DESCRIPTION
**Pull request by rmeertens**
@fvantienen Can you continue with this?

### CODE REVIEW

**Comment by fvantienen** [comment]
I will continue with this, but for simplicity sake I will start with adding the rotation to the buffer as well. Maybe even GPS coordinates for nicer picture loggings.

**Comment by flixr** [comment]
Is this still relevant with the merge of #1737 ?

**Comment by fvantienen** [comment]
Yes this is especially made for this, because you need to relate this image to a paparazzi compatible time in order to get a pose..

**Comment by flixr** [comment]
You should probably keep the timeval timestamp...

File: sw/airborne/modules/computer_vision/lib/v4l/v4l2.c [hunk 1]
```diff
@@ -92,8 +97,8 @@ static void *v4l2_capture_thread(void *data)
     assert(buf.index < dev->buffers_cnt);
 
     // Copy the timestamp
-    memcpy(&dev->buffers[buf.index].timestamp, &buf.timestamp, sizeof(struct timeval));
-
+    dev->buffers[buf.index].timestamp = buf.timestamp.tv_usec;
```
Comment (flixr) [inline sw/airborne/modules/computer_vision/lib/v4l/v4l2.c]: keeping timestamp as a timeval struct would make more sense than assigning it the remainder of the second in us...

Comment (rmeertens) [inline sw/airborne/modules/computer_vision/lib/v4l/v4l2.c]: Mmm, I think I agree with you. @fvantienen , what do you think? At first I thought you wanted to save tv_sec*1000000+tv_usec, but this gives us a flight time of 4294 seconds, just over an hour, before it overflows. 
Shall we just keep the timestamp struct?

Comment (flixr) [inline sw/airborne/modules/computer_vision/lib/v4l/v4l2.c]: get_sys_time_usec() also overflows after 72min, but that is normally not a problem...

**Comment by flixr** [comment]
Shall I change the timestamp back to the timeval struct or do you guys want to do it?
Just adding the time since startup in usec (pprz_timestamp) looks good to me...

**Comment by rmeertens** [comment]
I will do it next week, that ok?

**Comment by flixr** [comment]
sure

**Comment by rmeertens** [comment]
Thanks!!

**Pull request closed without merging**